strapi/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,34 @@
+// strapi service devcontainer
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.238.0/containers/typescript-node
+{
+	"name": "clic-website-devcontainer/strapi",
+	// Use docker-compose stack, relative to current file
+	"dockerComposeFile": "../../.devcontainer/docker-compose.yaml",
+	// Connect VSCode to following service defined in dockerComposeFile
+	"service": "strapi",
+	// Working directory inside devcontainer, where project is located
+	"workspaceFolder": "/app",
+	// Configure tool-specific properties.
+	"customizations": {
+		// Configure properties specific to VS Code.
+		"vscode": {
+			// Add the IDs of extensions you want installed when the container is created.
+			"extensions": [
+				// todo highlighter
+				"wayou.vscode-todo-highlight",
+				// linter
+				"dbaeumer.vscode-eslint",
+				// formatter
+				"esbenp.prettier-vscode"
+			]
+		}
+	},
+	// Comment out to connect as root instead. More info: https://aka.ms/vscode-remote/containers/non-root.
+	"remoteUser": "node",
+	// Use 'forwardPorts' to make a list of ports inside the container available locally.
+	// "forwardPorts": [],
+	// Use 'postCreateCommand' to run commands after the container is created.
+	// "postCreateCommand": "yarn install",
+	"postStartCommand": "python3 check_env.py"
```

Comment:
I like the idea of starting this through the devcontainer. However, we are limited by VSCode which cannot open multiple devcontainers at the same time, so for instance if someone opens the devcontainer of the NextJS service, this command will not run. I wish we'd have a better, unified scheme, but for now I think our best bet is to use the Dockerfile and inject the command directly: https://github.com/clicepfl/clic-website-v2/blob/10b0dee5aeb23b5be6e265f351904ba1a6a7cc85/strapi/.devcontainer/Dockerfile#L15 
We can use an `entrypoint.sh` script if it gets too crowded (see [entrypoint best practices](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#entrypoint))

Reply:
I've looked for librairies in JS, and I've found [joi](https://joi.dev/api/?v=17.7.0) which can both validate and generate values. I'll also use the [dotenv](https://github.com/motdotla/dotenv) as you said.